official/resnet/resnet_run_loop.py

```diff
@@ -122,7 +106,7 @@ def get_synth_input_fn(height, width, num_channels, num_classes):
     An input_fn that can be used in place of a real one to return a dataset
     that can be used for iteration.
   """
-  def input_fn(is_training, data_dir, batch_size, *args):  # pylint: disable=unused-argument
+  def input_fn(is_training, data_dir, batch_size, *args, **kwargs):  # pylint: disable=unused-argument,missing-docstring
```

Comment:
This function should be short enough to not require a docstring (< 10 lines); are you sure this extra disable is necessary?

Reply:
It was erroring on the Kokoro lint, but looks like it isn't now. It will forever remain a mystery.